eos-updater-avahi/docs/eos-updater-avahi.8

```diff
@@ -117,6 +117,13 @@ Describes the DNS\-SD record for Avahi to advertise to the local network. No
 advertisement will be made if this file is not present. This is the file which
 \fBeos\-updater\-avahi\fP adds, updates and removes when run.
 .\"
+.IP \fI/etc/avahi/services/eos\-ostree\-updater\-[0\-65535].service\fP 4
+.IX Item "/etc/avahi/services/eos-ostree-updater-[0-65535].service"
+Similarly to the eos-updater.service file in the same directory, it
```

Comment:
`eos-updater.service` needs to be formatted like other filenames in the man page, and the hyphen in it escaped.